services/userServices.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+const userDao = require('../models/userDao');
+const { axiosUser } = require('../util/axios');
+
+const loginService = async (kakaoToken) => {
+  try {
+    const user = await axiosUser(kakaoToken);
+    const id = user.data.id;
+    const image = user.data.properties.profile_image || null;
```

Comment:
변수 네이밍 수정하면 좋을 것 같습니다.

```suggestion
    const kakaoId = user.data.id;
    const profileImage = user.data.properties.profile_image || null;
```